config/aws-efs/manifests/aws-efs-csi-driver-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: aws-efs-csi-driver-operator
+channels:
+- name: stable
+  currentCSV: aws-efs-csi-driver-operator.v4.16.0
```

Comment:
Would you be willing to update all the v4.16.0 references to v4.17.0 in this PR?
This is for https://issues.redhat.com/browse/STOR-1606 -- we need to update all our OLM operators, but I don't want to file a separate PR for EFS while we're mid-transition to csi-operator.
See [this 4.16 commit](https://github.com/openshift/aws-efs-csi-driver-operator/commit/350bb7c47a2c6935f557124f0a1a1355c51404cb) for example.

Reply:
Good point, adding a separate commit with the version bump.